mautic/mautic
# All columns for agregators in  Reports

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR just add allow all columns use in aggregate columns. After update we are able to use AVG of unsubscribe ratio etc. 

![image](https://user-images.githubusercontent.com/462477/66134978-7b507a80-e5f9-11e9-8660-15aaf1a6bed0.png)

Also I noticed impossible use Page hit ID in Page Hits reports. 
This PR fixed it.

![image](https://user-images.githubusercontent.com/462477/66136438-c3709c80-e5fb-11e9-9d5c-a6b7df15e038.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1 Create emails type of report and group by something (category id for example)
2. Try set unsubscribe or unsubscribe ratio (for example SUM unsubscribed or AVG unsubscribe ratio)
3. You are not able to do that
4. Then try create page hits report. 
5. Try find ID of hit. You can use just ID (it's pages table ID)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. You are able to choose unsubscribe and unsubscribe ration for aggregator columns (for example SUM unsubscribed or AVG unsubscribe ratio)
3. Check if report works for you
4. Then try create page hits report
5. Try find ID of hit. You can use ID and Page hit ID
ID is landing page ID , and Page hit ID is hit ID
